website/server/controllers/api-v3/webhook.js

```diff
@@ -223,6 +223,10 @@ api.updateWebhook = {
 
     webhook.formatOptions(res);
 
+    // Tell Mongoose that the webhooks array has been modified
+    // so it actually commits the options changes to the database
+    user.markModified('webhooks');
```

Comment:
The fix looks good but this way it's going to send back to the database the entire array even if only one item was modified, it works also if you do `webhook.markModified('options');` this way it's telling mongoose that the Mixed type `options` object is modified. The issue happens both with arrays but also mixed types objects as mongoose doesn't know what's been modified, see https://mongoosejs.com/docs/schematypes.html#mixed

Reply:
Ah, okay. I've updated it to use `webhook.markModified('options')` instead. Also, I've updated the PR description to reflect the correction.

Reply:
thanks @robwhitaker ! The pr looks good and will be merged at the end of the week